### ISSUE TITLE
Blank Screen and Console Error when Accessing OHIF Viewer in Docker Setup

### ISSUE THREAD

**Issue by arizonal2014** [orthanc-server/orthanc-setup-samples#1]
I encountered an issue while setting up the OHIF viewer with Orthanc using the provided Docker setup from the repository. After following the instructions in the README file, I accessed "http://localhost/ohif/" but encountered a complete blank screen. Additionally, the console displayed the following error: "DevTools failed to load source map: Could not load content for http://localhost/ohif/html-parse-stringify.module.js.map: Unexpected token '<', "<!doctype "... is not valid JSON."

Steps to Reproduce:

Clone the repository: https://github.com/orthanc-server/orthanc-setup-samples/tree/master/docker/ohif
Follow the instructions in the README file.
Access "http://localhost/ohif/" in the browser.
Expected Behavior:
The OHIF viewer should load successfully, displaying the expected content.

Actual Behavior:
Upon accessing "http://localhost/ohif/", the browser window shows a blank screen, and the console displays an error regarding the source map.

Console Error:
DevTools failed to load source map: Could not load content for http://localhost/ohif/html-parse-stringify.module.js.map: Unexpected token '<', "<!doctype "... is not valid JSON.

Additional Information:

Accessing "http://localhost/orthanc/ui/app/#/" works as expected, indicating that the Orthanc UI is functioning properly.
Other steps mentioned in the README file, such as uploading a study in Orthanc and clicking on the OHIF viewer button, were followed accordingly.
Possible Cause and Suggestion:
It seems that there might be an issue with the configuration or loading of the OHIF viewer in the Docker setup. I suggest reviewing the configuration files and the way the viewer is being served through the nginx container. Additionally, it would be helpful to ensure that the necessary dependencies and resources for the OHIF viewer are properly included and accessible.

I appreciate your assistance in resolving this issue and enabling successful integration of the OHIF viewer with Orthanc. Please let me know if there is any further information or steps I can provide to help diagnose and resolve the problem.

**Comment by amazy** [comment]
Hi,

Just tried it (after I updated the setup to latest v3.6 version but it should have worked before)
![2023-06-14_10h40_55](https://github.com/orthanc-server/orthanc-setup-samples/assets/14176104/b536919b-870e-4430-a7eb-cf80c665eabe)

Note that this config is not supposed to show the study list in the OHIF viewer at `http://localhost/ohif/`, I removed the link from the readme.  Any help to enable the OHIF study list is welcome.

Best regards,

Alain.
